content/speakers/signup.md

```diff
@@ -1,5 +0,0 @@
----
-type: redirect
-redirectURL: 'https://docs.google.com/forms/d/e/1FAIpQLSeirnTIywKx5w3HVLny8pfZALUYmrqDOUTroGeUDESwqmrnSA/viewform?usp=sf_link'
```

Comment:
Removing this might break old links we might have posted around

Reply:
I don't think this particular one will. We posted about /apply . And even if it did the person would end up on a 404 page on our website and can likely navigate to the correct page.